amy/emails/tests/actions/test_admin_signs_instructor_up_for_workshop.py

```diff
@@ -181,15 +227,18 @@ class TestAdminSignsInstructorUpForWorkshopReceiverIntegration(TestBase):
     @override_settings(INSTRUCTOR_RECRUITMENT_ENABLED=True)
     @override_settings(FLAGS={"EMAIL_MODULE": [("boolean", True)]})
     @mock.patch("django.contrib.messages.views.messages")
-    @mock.patch("emails.actions.messages_action_scheduled")
+    @mock.patch(
+        "emails.actions.admin_signs_instructor_up_for_workshop"
+        ".messages_action_scheduled"
+    )
     def test_integration(
         self,
         mock_messages_action_scheduled: mock.MagicMock,
         mock_contrib_messages_views: mock.MagicMock,
     ) -> None:
         # Arrange
         host = Organization.objects.create(domain="test.com", fullname="Test")
-        person = Person.objects.create_user(
+        person = Person.objects.create_user(  # type: ignore
```

Comment:
Pylint has issues accessing correct manager for models if there's a custom manager used.

I'd still like to use this type checking because we have other models that have only the default manager.